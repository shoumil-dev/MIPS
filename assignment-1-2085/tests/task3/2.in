2
10
5
12
