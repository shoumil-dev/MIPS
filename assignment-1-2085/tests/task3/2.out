MARS 4.5  Copyright 2003-2014 Pete Sanderson and Kenneth Vollmar

Array length: Enter the value of hulk power: Enter num: Enter num: Hulk SMASH! >:(
Hulk Sad :(
Hulk smashed 1 people

